Tests/NIOSSHTests/SSHConnectionStateMachineTests.swift

```diff
@@ -57,12 +64,80 @@ final class SSHConnectionStateMachineTests: XCTestCase {
             }
 
             if clientBuffer.readableBytes > 0 {
-                server.bufferInboundData(&clientBuffer)
+                if dripFeed {
+                    while var next = clientBuffer.readSlice(length: 1) {
+                        server.bufferInboundData(&next)
+                        if clientBuffer.readableBytes > 0 {
+                            switch try assertNoThrowWithValue(server.processInboundMessage(allocator: allocator, loop: loop)) {
+                            case .some(.emitMessage(let message)):
+                                serverMessage.append(message)
+                            case .none:
+                                ()
+                            case .some(.noMessage):
+                                ()
+                            case .some(.possibleFutureMessage(let futureMessage)):
+                                waitingForServerMessage = true
```

Comment:
We need some caution here: can we `precondition` that `waitingForServerMessage` is `false` when we enter this block? The code doesn't handle it well.

Reply:
done!